docs/docusaurus/docs/conceptual_guides/gx_overview.md

```diff
@@ -0,0 +1,118 @@
+
+Installation and Setup -> Connect to data -> Create Expectations -> Validate data
+
+### Install Great Expectations
+
+GX is a library for <PythonVersion />, and can be installed using `pip`, a library installer bundled with Python installations.  Depending on your Source Data System, you may need to install GX with some additional dependencies (generally for SQL Source Data Systems, or environments like AWS).  We streamline this process by providing special `pip` commands for specific combinations of additional dependencies.
+
+For details on how to install GX and how to include any additional dependencies for your Source Data System, see [our installation guides](landing page: Setting up a GX environment->Installation and Dependencies).
+
+
+### Set up your GX Data Context
+
+A Great Expectations project stores its configurations in a **Data Context**.  In Python, the Data Context object also serves as your entry point for the GX API, allowing you to further configure and interact with GX.  Your Data Context manages various classes and helps limit the objects you need to directly manage to get GX working on your data.
+
+GX provides three types of Data Contexts:
+- An **Ephemeral Data Context**: This Data Context exists in memory, and will not persist beyond the current Python session.
+- A **Filesystem Data Context**: This Data Context exists as a folder and configuration files, allowing its contents to persist between Python sessions.
+- A **Cloud Data Context**: This Data Context also supports persistence between Python sessions, but additionally serves as the entry point for GX Cloud.
+
+<!-- For details on how to initialize and instantiate a Data Context, see [our Data Context guides](landing page: Setting up a GX environment->Data Contexts). -->
+
+
+#### Optional configurations for metadata Stores and Data Docs
+
+**Stores** are connectors to the metadata GX uses.  This includes things like:
+- The Datasource and Data Asset configurations that tell GX how to connect to your Source Data System.
+- The Expectations you have specified about your Source Data.
+- The Metrics that are recorded when GX validates certain Expectations against data.
+- The Checkpoints you have configured for validating your Expectations.
+- The Validation Results that GX returns when you run a Checkpoint.
+
+A GX Data Context comes with default configurations in place for your Stores.  By default, these exist in the same folder hierarchy as your Data Context (or in memory, if you are using an Ephemeral Data Context).  However, you can update these if you want to change where your Store configurations exist, such as hosting them in a shared cloud environment like AWS.
```

Comment:
```suggestion
A Data Context includes the default configurations for your Stores. Default configurations are stored with your Data Context, or in memory if you're using an Ephemeral Data Context. You can move your configurations to another storage location, or you can host them in a shared cloud environment such as AWS.
```
Very confused:

A **Data Context** includes default configurations for your Stores.  By default, **these exist in the same folder hierarchy as your Data Context** (or in memory, if you are using an Ephemeral Data Context). 

So, the Data Context includes default configurations and they are are stored with the Data Context? This needs to be clarified. I've made an attempt.